### PR TITLE
Implement Convert for Activations and Optimisers

### LINKED ISSUE #8: Add issue templates and pr template

### LINKED ISSUE #10: Provide Convert for the existing BH.oM.DeepLearning.Activations
Check https://github.com/BHoM/Keras_Toolkit/projects/1 to view the status of the converts.
This issue aims at ticking all the Activations column.

- [x] LeakyReLU.cs
- [x] LogSigmoid.cs
- [x] LogSoftmax.cs
- [x] ReLU.cs
- [x] Sigmoid.cs
- [x] Softmax.cs
- [x] Tanh.cs

### PR DESCRIPTION
**Pull request by epignatelli**
Closes #10 
Closes #8 

#### Change log
- Add Convert.FromKeras and Convert.ToKeras for:
  - LeakyReLU
  - LogSigmoid
  - LogSoftmax
  - ReLU
  - Sigmoid
  - Softmax
  - Tanh
  - Adam
  - SGD

#### Test file
https://burohappold.sharepoint.com/:f:/s/BHoM/EtM_cCW6kR9Jkan8fC-IS_MBFP9Ydurjkg9HU-K24yuLsg?e=8yqRuP

#### Additional comments
The Convert for `LogSoftmax` at this moment does not allow passing the `axis`/`Dimension` parameter through because of the way `tensorflow` treats the `tf.nn.log_softmax` object, which is passed a `function`

@AlaaAlfakara Would you mind start testing these things? I think it is useful if start familiarising with it, such that I can have feedback on what works and what not.

### CODE REVIEW

**Review by alelom — APPROVED** [review]
LGTM
image
